.github/actions/setup/action.yml

```diff
@@ -8,7 +8,7 @@ runs:
 
     - uses: actions/setup-node@v3
       with:
-        node-version: 14
+        node-version: 18
```

Comment:
our release workflows can also be updated to omit the extra setup-node steps:
https://github.com/Uniswap/interface/blob/2415a1e3cd6ba6cf3bab43188251bca2b56e5e45/.github/workflows/4-deploy-to-prod.yml#L77-L80

https://github.com/Uniswap/interface/blob/2415a1e3cd6ba6cf3bab43188251bca2b56e5e45/.github/workflows/2-deploy-to-staging.yml#L30-L33

https://github.com/Uniswap/interface/blob/2415a1e3cd6ba6cf3bab43188251bca2b56e5e45/.github/workflows/release.yaml#L101-L104

Reply:
Do we have a good way to dryrun test that these deploys still work?

Reply:
I think that a staging deploy could be considered a dry run.